cmake/tribits/doc/developers_guide/TribitsDevelopersGuide.rst

```diff
@@ -6304,6 +6304,41 @@ The following steps describe how to submit results to a CDash site using the
   .. the other cloned and updated by the CTest driver script.
 
 
+How to submit testing results to a custom CDash Group
+-----------------------------------------------------
+
+Following up on `How to submit testing results to a CDash site`_, to submit
+build and test results to a custom "Group" on CDash (instead of just
+"Nightly", "Continuous" or "Experimental"), one just has to create the new
+group on CDash using the CDash GUI interface and then tell the ctest -S local
+driver to submit results to that CDash group.  The steps for doing this are
+given below.
+
+1) Create the new CDash group ``<special_group>`` for CDash project
+``<ProjectName>`` on the targeted CDash site.
+
+  If the CDash group ``<special_group>`` is not already created, then one can
+  create it by first logging into CDash with an account that can modify the
+  CDash project ``<ProjectName>``.  Once logged in, go to the project edit
+  page and select "Settings" and "Groups".  From there, create the new group
+  ``<special_group>``.  Set the "Build Type" to either "Daily" or "Latest".
+
+2) Set ``${PROJECT_NAME}_TRACK=<special_group>`` with the CTest -S driver
+script.
+
+  One can either do that by setting ``SET(${PROJECT_NAME}_TRACK
+  <special_group>)`` in the CTest -S ``*.cmake`` driver script itself or can
+  set it in the environment when running the ctest -S driver script.  For
+  example::
+
+    $ env <Project>_TRACK=<special_group> ... \
+      ctest -V -S <ctest_driver>.cmake
+
+  If the "build type" for the CDash group ``<special_group>`` was set to
+  "Daily", then set `CTEST_TEST_TYPE`_ to ``Nightly``.  Otherwise,
+  ``CTEST_TEST_TYPE`` can be set to ``Continuous`` or ``Experimental``.
```

Comment:
I don't think this is valid reStructuredText for enumerated lists.  I think you'll need
```
1) Create the new CDash group ``<special_group>`` for CDash project
   ``<ProjectName>`` on the targeted CDash site.

   If the CDash group ``<special_group>`` is not already created, then one can
   create it by first logging into CDash with an account that can modify the
   etc.
```

for it to get parsed correctly.  Note the indentation lining up with the `C` in `Create`.

Reply:
@jmgate, this is addressed in commit https://github.com/TriBITSPub/TriBITS/commit/38cc5ab92224df62d9bce8617249ff1ba1edd3fb and you can see the updated formatting at:

* https://tribits.org/doc/TribitsDevelopersGuide.html#how-to-submit-testing-results-to-a-custom-cdash-group

Thanks for noting this.